src/PresentationalComponents/TableToolbar/TableToolbar.js

```diff
@@ -40,7 +45,8 @@ TableToolbar.propTypes = {
     isFooter: propTypes.bool,
     results: propTypes.number,
     children: propTypes.any,
-    className: propTypes.string
+    className: propTypes.string,
+    selected: propTypes.number
```

Comment:
This should be number. Or `oneOfType([ propTypes.number, propTypes.string])`

Reply:
@karelhala what should be number? `selected` is a number

Reply:
Yeah, my bad. I saw string.